docs/apps/rbac/for_clients.md

```diff
@@ -0,0 +1,114 @@
+## Using DAB RBAC as an API Client
+
+This section tells how to use the API endpoints as a client;
+what requests to make and how to build request data.
+
+You need a server running a Django project that uses this system.
+Use test_app in this repo for a demo, see `test_app/README.md` for bootstrap.
+http://127.0.0.1:8000/admin/ allows login as admin user (password is "password").
```

Comment:
I think after your #471 , password will be "admin"

Reply:
Good call-out. This whole thing is out-of-date, and I'll update it.